server/index.js

```diff
@@ -1,9 +1,32 @@
 const express = require('express');
 const path = require('path');
+const bodyparser = require('body-parser');
 const db = require('../data/index');
 
+
 const app = express();
 
 const port = process.env.PORT || 3000;
 
+app.use('/rooms/:room_id', express.static(path.join(__dirname, '..', 'client', 'dist')));
```

Comment:
@Pfdiamond13 @fradleybrancis This line might cause some trouble due to the way browsers request static assets.